views/account/profile/index.ejs

```diff
@@ -99,6 +99,56 @@
               >
                 Save Changes
               </button>
+              <!-- Button to trigger the help modal -->
+              <button
+                id="modal-btn"
+                class="btn btn-outline-secondary btn-m"
+                type="button"
+                aria-label="information"
+              >
+                <i id="info-button" class="fa-solid fa-circle-question"></i>
+              </button>
+              <!-- Help modal -->
+              <div class="helpModal" aria-label="help modal">
+                <div class="helpModal" aria-label="help modal">
+                  <div class="modal">
+                    <div class="modal-content">
+                      <span class="close-btn">&times;</span>
```

Comment:
This should be a color other than white when you hover over it. I think it blending it can be harder to see.

Reply:
do you mean the "x" closing button, like change its color when its hovered over? i was debating just getting rid of the hover that triggers the color change or i can do red. red is a pretty common color for the exit button.

Reply:
<img width="572" alt="Screen Shot 2022-11-28 at 2 16 52 PM" src="https://user-images.githubusercontent.com/57613959/204362156-167a27f3-eccf-43e3-b871-7cc89c261783.png">

is this ugly, or should i just keep it black all the time and remove the hover feature?

Reply:
I think you should keep the hover since every other input element has a hover feature to them. What I would recommend is to make the close modal button a darker red, and on hover, make it a lighter red. It shouldn't be a dark red like crimson but the idea is that the button should "brighten up" when the user hovers or clicks on it.

Reply:
thats a great idea, i'll do that right now. thanks sterly!

Reply:
i might see if there's an outline feature that can be added to the red so the red pops out more. right now, the two shades of red still look the same

Reply:
https://getbootstrap.com/docs/5.0/components/close-button/ you can also look at this for reference. You should just be able to copy the code into the application.

Reply:
You'd probably have to add btn-danger as a class to make it red

Reply:
also if you were wondering why i didn't go with the red "X" button its because i used the bootstrap one instead like you advised and it looks better, but i couldnt change the color as it was an image not a button. but it looks good the way it is now too so no worries!